src/pytest_bdd/scenario.py

```
@@ -384,7 +385,7 @@ def scenario(
         scenario = feature.scenarios[scenario_name]
     except KeyError:
         feature_name = feature.name or "[Empty]"
-        raise exceptions.ScenarioNotFound(
+        raise exceptions.ScenarioNotFound(  # noqa: B904
```

we should do `raise ... from None` instead of suppressing the error from ruff

---

Cheers! Applied